iOS/SceneDelegate.swift

```diff
@@ -72,6 +74,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         return scene.userActivity
     }
 
+    func configureStyle(for style: UIUserInterfaceStyle) {
```

Comment:
I would prefer to see the `Theme` type passed as a parameter instead of the `UIUserInterfaceStyle`